modules/standard/ChapelIO.chpl

```diff
@@ -199,13 +66,33 @@ module ChapelIO {
       return ret;
     }
 
+    pragma 'fn exempt instantiation limit'
+    @chpldoc.nodoc
+    proc serializeDefaultImpl(writer:fileWriter, ref serializer,
+                              const x:?t) throws where isUnionType(t) {
+      // Handle unions.
+      // print out just the set field for a union.
```

Comment:
Do we have a test of serializing unions already?  If not, I think this means we should add one